docker-compose_production.yml

```diff
@@ -260,6 +272,13 @@ volumes:
       device: ${DOCKER_MIGRID_ROOT}/state
       o: bind
   
+  state-ro:
+    driver: local
+    driver_opts:
+      type: none
+      device: ${DOCKER_MIGRID_ROOT}/state/vgrid_files_writable
+      o: bind
```

Comment:
I think it's better if we add the read only option here so that the volume it self is read only.
Otherwise a container could mount it writeable due to a config error.

Reply:
with the new change its reported as:
```
{
                "Type": "volume",
                "Name": "docker-migrid_state-ro",
                "Source": "/var/lib/docker/volumes/docker-migrid_state-ro/_data",
                "Destination": "/home/mig/state/vgrid_files_readonly",
                "Driver": "local",
                "Mode": "z",
                "RW": false,
                "Propagation": ""
            },
```